_dashboards/management/scheduled-query-acceleration

```diff
@@ -0,0 +1,228 @@
+
+### Better Control Over Refresh Scheduling
+
+With SQA, you have more flexible scheduling options. You can set refresh intervals based on your specific requirements, ensuring efficient data management without overburdening driver nodes. This means greater control over resource usage and refresh frequency that better aligns with your needs.
+
+### Simplified Index Management
+
+SQA also simplifies index management by allowing you to adjust index settings without multiple, complicated queries. You can easily update refresh intervals or other parameters, thereby streamlining your workflow and reducing manual effort.
+
+## Getting Started
+
+### Prerequisites
+
+- **OpenSearch Version Requirements**: OpenSearch 2.17 or later.
+- **Required Plugins and Dependencies**: SQL plugin.
+- **System Requirements**: EMR Serverless, Amazon S3.
+
+### Recommended Reading
+
+- [Materialized Views / Cached Views / Scheduled Indexing (MV / CV / SI)](https://opensearch.org/docs/latest/dashboards/management/accelerate-external-data/)
+- [Scheduled Refresh vs. Continuous Streaming](https://github.com/opensearch-project/opensearch-spark/blob/main/docs/index.md#flint-index-refresh)
+- [Index State Management](https://github.com/opensearch-project/opensearch-spark/blob/main/docs/index.md#flint-index-refresh)
+
+## Configuration
+
+### Cluster Settings
+
+To enable and configure SQA for efficient query management and cost-effective resource utilization, two essential cluster settings need to be adjusted:
+
+#### Enable Async Query Execution
+
+This setting allows users to enable the asynchronous query execution feature, which is crucial for handling queries in a non-blocking manner. Enabling async query execution improves query efficiency and resource allocation by allowing tasks to run in the background without tying up system resources.
+
+[Setting Link](https://github.com/opensearch-project/sql/blob/main/docs/user/admin/settings.rst#pluginsqueryexecutionengineasync_queryenabled)
+
+#### Configure Async Query External Scheduler Interval
+
+This setting specifies the interval at which the external scheduler checks for tasks, allowing users to customize how often the scheduler initiates refresh operations. Adjusting this interval based on workload requirements can optimize resource usage and control the frequency of driver node operations, helping to manage costs effectively.
+
+[Setting Link](https://github.com/opensearch-project/sql/blob/main/docs/user/admin/settings.rst#pluginsqueryexecutionengineasync_queryexternal_schedulerinterval)
+
+### Spark Configurations
+
+- `spark.flint.job.externalScheduler.enabled`: Default is `false`. Enable external scheduler for Flint auto-refresh to schedule refresh jobs outside of Spark.
```

Comment:
on sql plugin 2.17 this value is passsed as true by default

Reply:
So is the default for `spark.flint.job.externalScheduler.enabled` true then?